src/components/ContactDetails.vue

```diff
@@ -338,6 +344,7 @@ export default {
 			pickedAddressbook: null,
 
 			contactDetailsSelector: '.contact-details',
+			excludeFromBirthdayKey: 'x-nc-exclude-from-birthday-calendar',
```

Comment:
iCalendar property parameters are usually in upper case, so let's be explicit here, even if the vCard serializer might fix it anyway.
```suggestion
			excludeFromBirthdayKey: 'X-NC-EXCLUDE-FROM-BIRTHDAY-CALENDAR',
```

Reply:
I revert the change to set it lowercase, it seems that the ICAL.Component expose the properties as lowercase.

They are still sent to server as uppercase